utils/src/main/java/com/cloud/utils/UriUtils.java

```diff
@@ -200,37 +198,27 @@ public static String getUpdateUri(String url, boolean encrypt) {
     }
 
     // Get the size of a file from URL response header.
-    public static Long getRemoteSize(String url) {
-        Long remoteSize = (long)0;
+    public static long getRemoteSize(String url) {
```

Comment:
@marcaurele I think this needs to be changed back to Long, or else the method cannot be found as it is called with type Long.